fcc-test-suite-template-master 2/index.html

```diff
@@ -8,25 +8,35 @@
 
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    "Hello World, we are software developers!"
     <main id="main">
-      <h2 id="title">Dr. Norman Borlaug</h2>
-      <div id="img-div" style="display: flex; flex-direction: column; ">
+      <h1>"Hello World, we are software developers!"</h1>
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div" style="display: flex; flex-direction: column">
         <img
           src="https://images.unsplash.com/photo-1573496130141-209d200cebd8?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2338&q=80"
           alt=""
           id="image"
-          style="max-width: 100%; align-self: center;"
```

Comment:
Again same here, kindly add the reason explaining why this person shouldn't use inline styling? 👍

Reply:
As previously requested, you are supposed to state to the student why this change is being requested in order to make it easier for them to implement it as expected.

INSTEAD OF

```
Oops! You have a duplicate inline styling here, kindly remove one.
Check Line 19 and 20.
```

SAY

```
Oops! You have a duplicate inline styling here, kindly remove one. Doing so will make your codebase less repetitive and cleaner.

Check Line 19 and 20.
```